src/components/shared/Validations.tsx

```diff
@@ -28,8 +33,30 @@ const numberValidationFormatters: Record<string, (value: unknown) => string> = {
   minLength: value => `>= ${value} characters`,
   maximum: value => `<= ${value}`,
   exclusiveMaximum: value => `< ${value}`,
-  maxItems: value => `< ${value} items`,
-  maxLength: value => `< ${value} characters`,
+  maxItems: value => `<= ${value} items`,
+  maxLength: value => `<= ${value} characters`,
+};
+
+const validationsFormatter = (name: string) => (value: unknown[] | unknown): ValidationFormat | null => {
+  const values = Array.isArray(value) ? value : [value];
+  if (values.length) {
+    return {
+      name: values.length > 1 ? `${name} values` : `${name} value`,
+      values: values.map(stringifyValue),
+    };
+  }
+  return null;
+};
+
+const validationFormatters: Record<string, (value: unknown) => ValidationFormat | null> = {
+  ['const']: validationsFormatter('Allowed'),
+  enum: validationsFormatter('Allowed'),
+  examples: validationsFormatter('Example'),
+  example: validationsFormatter('Example'),
+  ['x-example']: validationsFormatter('Example'),
```

Comment:
I'm not sure if json-schema-tree supports this. You might want to verify it.

Reply:
It might not. Same with `const` so I took it from `fragment` to not update JST but we might want to do that. Though not sure if `x-example` is worth it.

Reply:
>  so I took it from fragment to not update JST

We shouldn't be using `fragment`. :grimacing: